src/Plugin_Toggler.php

```diff
@@ -142,6 +165,42 @@ public function ajax_toggle_plugin_version() {
 		die();
 	}
 
+	/**
+	 * Retrieves the controls.
+	 *
+	 * @return string The HTML to use to render the controls.
+	 */
+	public function get_controls() {
+		$output  = '<h2>Plugin toggler</h2>';
+		$output .= '<form action="' . esc_url( admin_url( 'admin-post.php' ) ) . '" method="POST">';
+		$output .= wp_nonce_field( 'plugin_toggler', '_wpnonce', true, false );
+		$output .= '<input type="hidden" name="action" value="yoast_seo_plugin_toggler">';
+
+		$output .= '<input type="checkbox" ' . checked( $this->options['plugin_toggler'], true, false ) . ' name="plugin_toggler" id="plugin_toggler"/> <label for="plugin_toggler">Show plugin toggler.</label>';
+		$output .= '<br/><br/>';
+		$output .= '<button class="button button-primary">Save</button>';
+		$output .= '</form>';
+
+		return $output;
+	}
+
+	/**
+	 * Handles the form submit.
+	 *
+	 * @return void
+	 */
+	public function handle_submit() {
+		if ( check_admin_referer( 'plugin_toggler' ) !== false ) {
+			$this->options['plugin_toggler'] = false;
```

Comment:
This can be simplified to `$this->options['plugin_toggler'] = isset( $_POST['plugin_toggler'] );`